src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Main.cs

```diff
@@ -73,8 +90,29 @@ private List<Result> Commands()
                     IcoPath = $"Images\\shutdown.{IconTheme}.png",
                     Action = c =>
                     {
-                        Helper.OpenInShell("shutdown", "/s /t 0");
-                        return true;
+                        if (this._context.CurrentPluginMetadata.ConfirmSys)
+                        {
+                            MessageBoxResult messageBoxResult = MessageBox.Show(
+                                "You are about to shut down this computer, are you sure?",
```

Comment:
Messages that are visible to users should be added to resources. Take a look at `Properties/Resources.resx`. It is needed for localization

Reply:
Done